src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/params.py

```diff
@@ -586,6 +599,30 @@ def _configure_security_policy_storage_params(cmd):
 
 
 #####
+#           sql server aad-admin
+######
+
+
+with ParametersContext(command='sql server aad-admin create') as c:
+    c.expand('properties', ServerAzureADAdministrator)
+    c.argument('login',
+               options_list=('--user', '-u'))
```

Comment:
Not in all the cases. For example line 510. The original reason to break lines is that of the line length limit was 100. Now the line limit is 120 and additionally, in this case, there is sufficient space to have parameters in the same line. 

The horizontal real estate is important, so as of vertical. If you can put more lines of code on one page it will save the code reader a lot of time from scrolling back and forth. It is up to you eventually since this is not a mandatory code style rule we enforced currently. But I highly recommend you to save some space.